setup.py

```diff
@@ -513,6 +524,17 @@ def main(args):
         if len(args)>=4:
             minifier = args[3]
         install_html5(install_dir, minifier)
+        if len(args)<3:
+            # Move and symlink default-settings.txt
+            os.makedirs(os.path.normpath(os.path.join(sys.prefix, "../etc/xpra/html5-client")), exist_ok=True)
+            default_settings_files = { "default-settings.txt", "default-settings.txt.gz", "default-settings.txt.br" }
```

Comment:
As above.
Simply not having the `.gz` and `.br` is cleaner.

Reply:
Will fix